lib/baseline.py

```diff
@@ -7,7 +7,7 @@
 import json
 import os
 import slash
-from common import get_media
+from .common import get_media
```

Comment:
This change is in the wrong commit.  Please combine all "import" syntax changes into a single commit, together.